lib/watcher/express-records.js

```diff
@@ -0,0 +1,10 @@
+var agent = require('../agent');
+
+exports.worker = function(handle) {
+  agent().on('express:usage-record', function(record) {
+    handle.send({
+      cmd: 'express:usage-record',
+      record: record
```

Comment:
compare with status-wd below, and you'll see that express-records don't have startTime/pst. This is not changed, did you miss this, or was it just not necessary?

I suspect we could do some cleanup of the message formats, to ensure consistent `pid/wid/pst` was present in all the notifications, but that wouldn't be backwards compat, so I'm reluctant to do it in a PR that is otherwise pure-refactor.